pkg/storage/allocator.go

```diff
@@ -55,6 +55,23 @@ var (
 	// making decisions based on them.
 	// Made configurable for the sake of testing.
 	MinLeaseTransferStatsDuration = time.Minute
+
+	// EnableLeaseRebalancing controls whether lease rebalancing is enabled or
+	// not. Exported for testing.
+	EnableLeaseRebalancing = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_LEASE_REBALANCING", true)
+
+	// EnableLoadBasedLeaseRebalancing controls whether lease rebalancing is done
+	// via the new heuristic based on request load and latency or via the simpler
+	// approach that purely seeks to balance the number of leases per node evenly.
+	EnableLoadBasedLeaseRebalancing = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_LOAD_BASED_LEASE_REBALANCING", false)
+
+	// LeaseRebalancingAggressiveness enables users to tweak how aggressive their
+	// cluster is at moving leases towards the localities where the most requests
+	// are coming from. Lower settings than 1.0 will be less aggressive about
+	// moving leases toward requests than the default, while higher settings will
+	// cause more aggressive placement.
+	// Setting this to 0 effectively disables load-based lease rebalancing.
+	LeaseRebalancingAggressiveness = envutil.EnvOrDefaultFloat("COCKROACH_LEASE_REBALANCING_AGGRESSIVENESS", 1.0)
```

Comment:
`> 1` is definitely allowed (encouraged, even), so apparently I need to clarify this comment :)

Ensuring users don't go ` < 0` would be wise, though. Done.